src/libs/ReportUtils.ts

```diff
@@ -3727,7 +3727,7 @@ function buildOptimisticAddCommentReportAction(
     }
 
     const isAttachment = !text && file !== undefined;
-    const attachmentInfo = file ?? {};
+    const attachmentInfo = file ? {...file} : {};
```

Comment:
Should we add a comment since it's not immediately obvious why we're doing this here or why `??` wouldn't work?